src/framework/preferences/css/stylus/PrefsEditor.styl

```diff
@@ -122,21 +122,65 @@
             display:block;
             top:0;
             left:0;
-            height: 1.5em;
-            width: 1.5em;
             margin-top: -0.4em;
             margin-left: -0.4em;
-            background-color: #fff;
-            border: 1px solid #b3b3b3;
-            border-radius: 2em;
-            box-shadow: 4px 2px 3px rgba(0,0,0,0.3);
-            background-image: linear-gradient(right top, rgb(255,255,255) 46%, rgb(233,234,234) 73%);
+            slider-thumb-common()
```

Comment:
Will do - I'm uncertain how stable / active it is, but is it worth considering the use of [grunt-stylint](https://github.com/xdissent/grunt-stylint) in the future?